public/app/routes/routes.tsx

```diff
@@ -208,6 +208,12 @@ export function getAppRoutes(): RouteDescriptor[] {
       path: '/admin/access',
       component: () => <NavLandingPage navId="cfg/access" />,
     },
+    {
+      path: '/admin/access/groupsync',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "GroupSync" */ 'app/extensions/groupsync/GroupSyncEditor')
+      ),
+    },
```

Comment:
I think we can also put this behind a feature toggle, similarly as what has been done for SSO settings ([example](https://github.com/grafana/grafana/blob/v11.2.0/public/app/routes/routes.tsx#L302-L310)).

Reply:
Definitely, added here (and added the access check in the `roles` option): 56e752a383f